src/components/header/header/header.ts

```diff
@@ -170,11 +176,16 @@ export class SbbHeaderElement extends LitElement {
     }
   }
 
+  private _addFocusableElementsListeners(): void {
+    this._focusHandler.disconnect();
```

Comment:
I know it's correct but a little bit unreadable that in disconnect, it already sets a new abort controller. It feels more like now it's lost :P